python/pandemic_simulator/environment/regulation_stages.py

```diff
@@ -102,86 +115,106 @@
 # could reopen, and free movement was granted to all citizens within their Region; movement across Regions was still
 # banned for non-essential motives. Furthermore, on 25 May swimming pools and gyms could also reopen, and on 15 June
 # theatres and cinemas.
-italian_regulations: List[PandemicRegulation] = [
-    PandemicRegulation(stage=0),
-    PandemicRegulation(stay_home_if_sick=True,
-                       practice_good_hygiene=True,
-                       wear_facial_coverings=False,
-                       social_distancing=0.2,
-                       location_type_to_rule_kwargs={
-                           Office: {'lock': False},
-                           School: {'lock': False},
-                           BarberShop: {'lock': False},
-                           RetailStore: {'lock': False},
-                       },
-                       stage=1),
-    PandemicRegulation(stay_home_if_sick=True,
-                       practice_good_hygiene=True,
-                       wear_facial_coverings=False,
-                       social_distancing=0.25,
-                       location_type_to_rule_kwargs={
-                           Office: {'lock': False},
-                           School: {'lock': True},
-                           BarberShop: {'lock': False},
-                           RetailStore: {'lock': False},
-                       },
-                       stage=2),
-    PandemicRegulation(stay_home_if_sick=True,
-                       practice_good_hygiene=True,
-                       wear_facial_coverings=True,
-                       social_distancing=0.6,
-                       risk_to_avoid_gathering_size={Risk.HIGH: 0, Risk.LOW: 0},
-                       location_type_to_rule_kwargs={
-                           Office: {'lock': False},
-                           School: {'lock': True},
-                           BarberShop: {'lock': True},
-                           RetailStore: {'lock': True},
-                       },
-                       stage=3),
-    PandemicRegulation(stay_home_if_sick=True,
-                       practice_good_hygiene=True,
-                       wear_facial_coverings=True,
-                       social_distancing=0.8,
-                       risk_to_avoid_gathering_size={Risk.HIGH: 0, Risk.LOW: 0},
-                       location_type_to_rule_kwargs={
-                           Office: {'lock': True},
-                           School: {'lock': True},
-                           BarberShop: {'lock': True},
-                           RetailStore: {'lock': True},
-                       },
-                       stage=4)
+italian_regulations: List[CovidRegulation] = [
```

Comment:
Above comment applies here too

Reply:
Also the comment on open times for Italy applies here too - (@rcapobianco can you confirm?)